<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vital Tracker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="ecgLayout">
      <item>
       <widget class="QLabel" name="labelEcg">
        <property name="text">
         <string>ECG</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WaveformView" name="ecgView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="spo2Layout">
      <item>
       <widget class="QLabel" name="labelSpo2">
        <property name="text">
         <string>SpOâ‚‚</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WaveformView" name="spo2View"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="respLayout">
      <item>
       <widget class="QLabel" name="labelResp">
        <property name="text">
         <string>Respiration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WaveformView" name="respView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="bpLayout">
      <item>
       <widget class="QLabel" name="labelBp">
        <property name="text">
         <string>Blood Pressure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WaveformView" name="bpView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionMute"/>
   <addaction name="actionTimestamp"/>
  </widget>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionTimestamp">
   <property name="text">
    <string>Timestamp</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveformView</class>
   <extends>QWidget</extends>
   <header>waveformview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>


